providers/NMSReflectionProvider/src/main/java/net/ess3/nms/refl/providers/ReflFormattedCommandAliasProvider.java

```diff
@@ -0,0 +1,52 @@
+package net.ess3.nms.refl.providers;
+
+import net.ess3.nms.refl.ReflUtil;
+import net.ess3.provider.FormattedCommandAliasProvider;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.FormattedCommandAlias;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class ReflFormattedCommandAliasProvider extends FormattedCommandAliasProvider {
+
+    protected final Class<? extends FormattedCommandAlias> formattedCommandAliasClass;
+    private final Field formatStringsField;
+
+    @SuppressWarnings("unchecked")
+    public ReflFormattedCommandAliasProvider() {
+        Class<? extends FormattedCommandAlias> formattedCommandAliasClass = null;
+        Field formatStringsField = null;
+        try {
+            formattedCommandAliasClass = (Class<? extends FormattedCommandAlias>) ReflUtil.getOBClass("command.FormattedCommandAlias");
```

Comment:
Why are you using reflection here, let alone on a class already in the classpath?

Reply:
I use reflection here, because the method `FormattedCommandAlias#buildCommand(String, String[])` has private access, which we need to determine the commands which are being pointed to by this alias (and thus apply limitations / cooldowns on those).

Reply:
Should have been more clear, I meant why are you getting the class with reflection? You can just use `FormattedCommandAlias.class`

Reply:
Ah right, good call, will remove it.